R/tm_a_pca.R

```diff
@@ -1,4 +1,4 @@
-#' Principal component analysis module
+#' teal Module: Principal Component Analysis
```

Comment:
- Are we bringing back title case?
  - We've been avoiding them across all other `{teal.*}` packages. `{tmc}` is the outlier
- Backticks on `` `teal` ``?

```suggestion
#' `teal` module: Principal component analysis
```

or 

```suggestion
#' `teal` module: Principal Component Analysis
```

Reply:
I get it that we want to make it similar to `{tmc}`, but maybe we should only add the prefix and keep the same styling otherwise.

And suggest this to `{tmc}`

Reply:
@vedhav maybe you can also do a standarization in `{tmc}` while you are on a cleaning spree of titles in here?

Reply:
Sure thing

Reply:
Perhaps we can involve the sme team in the discussion before we decide on a format?

Reply:
yes. what about starting a discussion in this repository and CCing @shajoezhu @Melkiades and @edelarua

Reply:
Started the discussion [here](https://github.com/insightsengineering/teal.modules.clinical/discussions/1086)